deep_utils/utils/download_utils/download_utils.py

```diff
@@ -73,7 +73,8 @@ def download_file(
                         downloaded += len(data)
                         f.write(data)
                         done = int(50 * downloaded / total)
-                        sys.stdout.write("\rDownloading: [{}{}]".format("█" * done, "." * (50 - done)))
+                        sys.stdout.write("\rDownloading {}: {}% [{}{}]"
+                                         .format(file_name, round((downloaded*100/total),2), "█" * done, "." * (50 - done)))
```

Comment:
please follow pep8
put comma before 2 and after comma
Thanks bro

Reply:
changes made, please review